modules/cassandra/metrics.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+package cassandra
+
+type metrics struct {
+	throughput   *THROUGHPUT         `stm:"throuput"`
+	latency      *LATENCY            `stm:"latency"`
+	cache        *CACHE              `stm:"cache"`
+	disk         *DISK               `stm:"disk"`
+	gc_count     *GARBAGE_COLLECTION `stm:"java_gc_count"`
```

Comment:
- use camelCase, not snake_case.
- do not use UPPER CASE.
- all the structure/vars names should start with a lowercase letter.

Reply:
Done.